#goal definition
geometry_msgs/PoseStamped target_pose
---
#result definition
---
#feedback definition
geometry_msgs/Pose actual_pose
string stat
